brands/amenity/veterinary.json

```diff
@@ -8,5 +8,16 @@
       "brand:wikipedia": "en:Banfield Pet Hospital",
       "name": "Banfield Pet Hospital"
     }
+  },
+  "amenity/veterinary|VCA Pet Hospital": {
```

Comment:
Make sure to match the name tag here "VCA Pet Hospital" -> "VCA Animal Hospital"

Reply:
hah you beat me to it

Reply:
I realized right as I opened the PR that I messed that up